backend/src/main/kotlin/cz/loono/backend/Application.kt

```diff
@@ -19,7 +20,7 @@ import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
 @EnableScheduling
 class Application
 
-//change to trigger github action
+// change to trigger github action
```

Comment:
We don't this comment, don't we?

Reply:
neni primergovany master, nebude tam :-)